defaults/severities.yaml

```diff
@@ -84,3 +84,5 @@ NodetoolEvent: ERROR
 ScyllaBenchEvent: CRITICAL
 CassandraStressEvent: CRITICAL
 GeminiStressEvent: CRITICAL
+ScyllaServerEvent: NORMAL
```

Comment:
Let's call it `ScyllaServerStatusEvent` or `ScyllaServiceStatusEvent`

Reply:
ScyllaServerStatusEvent it is :)